src/shogun/preprocessor/RandomFourierGaussPreproc.cpp

```diff
@@ -45,7 +44,8 @@ void RandomFourierGaussPreproc::init_basis(int32_t dim_input_space)
 void RandomFourierGaussPreproc::fit(std::shared_ptr<Features> f)
 {
 	auto num_features = f->as<DenseFeatures<float64_t>>()->get_num_features();
-	require(num_features > 0, "Input space dimension must be greater than zero");
+	require(num_features > 0, "Given input space dimension {num_features} is less than zero", num_features);
```

Comment:
this will not render correctly i think. Could you pls run the code and look at the error msg...have a look at other error msgs to get the syntax of our require method

Reply:
```suggestion
	require(num_features > 0, "Given input space dimension {} is less than zero", num_features);
```

Reply:
Also "Given input space dimension {} must be positive."

Reply:
Thanks for your input @LiuYuHui !